core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -219,9 +219,9 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
         <div class="oppia-learner-dashboard-main-content-container">
           <div ng-if="activeSection === LEARNER_DASHBOARD_SECTION_I18N_IDS.INCOMPLETE && activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.EXPLORATIONS">
             <span style="display: block; padding: 5px 7.5px;"
-                  ng-if="numberDeletedIncompleteExplorations !== 0">
-              <span translate="I18N_LEARNER_DASHBOARD_DELETED_INCOMPLETE_EXPLORATIONS"
```

Comment:
Is `I18N_LEARNER_DASHBOARD_DELETED_INCOMPLETE_EXPLORATIONS` and the equivalent in the other five states still in use anywhere else? If not, should we remove it from the i18n files?

Reply:
This is also available in other 5-6 languages, Do I have to delete everywhere?
I'm new with angular-translation and it's hard to pass tests after deleting it from en.json and qqq.json.

Reply:
If a key is not in use, it's fine to delete it from all i18n files that use it.

Reply:
Done!